src/sqlittaaja/config.py

```diff
@@ -33,13 +33,30 @@ def parse(self, config: dict[str, Any]):
         match config.get("exercises"):
             case list(exercises_section):
 
-                def get_exercise(exercise) -> (str, str):
+                def get_exercise(exercise) -> (str, str, list[str], list[str]):
```

Comment:
We have some code duplication here with `must_contain` and `must_not_contain`. Could you perhapse put them here before the match?